ncm-fstab/src/main/perl/fstab.pm

```diff
@@ -75,26 +119,34 @@ sub mount_from_entry
 # fstab.
 sub delete_outdated
 {
-    my ($self, $fstab, %mounts) = @_;
+    my ($self, $fstab, $mountsr, $remove) = @_;
 
     my @rm;
-
+    my %mounts = %$mountsr;
     seek($fstab, 0, SEEK_SET);
 
     while (my $f = <$fstab>) {
     	my $mount = $self->mount_from_entry($f) or next;
-    	if (!exists ($mounts{$mount})) {
+    	if (!exists ($mounts{$mount}) && !exists($mounts{abs_path($mount)})) {
     	    CAF::Process->new ([UMOUNT, $mount],
     			       log => $self)->run();
     	    push (@rm, $f);
-    	    $self->verbose ("Scheduling for removal: $mount");
+            $self->verbose ("Scheduling for removal of fstab: $mount");
```

Comment:
Are these to log messages not a bit redundant and confusing? Could they be combined into a single message? It would also be great to log when the umount was successful or not according to $?